models/article.js

```diff
@@ -3,8 +3,13 @@ const db = require("../db/connection");
 exports.selectArticles = async () => {
   const articles = await db.query(
     `
-    SELECT article_id, title, topic, author, created_at, votes 
-    FROM articles 
+    SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes,
+    CAST(COUNT(comments.article_id) AS INT)
```

Comment:
Nice use of CAST to produce numbers rather than string representations of the comment_count